File: context-tree/tests/fixtures/languages/java/Sample.java
// A fixture for Java extraction tests.
package com.example;

import java.util.List;
import java.util.ArrayList;

/** Sample is the fixture container class. */
public class Sample {
    private int global = 10;

    /**
     * Return the sentinel CTX_SENTINEL_JAVADOC_8f1b marker.
     *
     * A second doc line so a later --doc render has more than one line.
     */
    public int value() {
        return global;
    }

    public int render() {
        List<Integer> items = new ArrayList<>();
        return value() + items.size();
    }
}

File: context-tree/tests/fixtures/languages/kotlin/Sample.kt
// A fixture for Kotlin extraction tests.
package com.example

import kotlin.math.PI

/**
 * value returns the sentinel CTX_SENTINEL_KDOC_7a2c.
 *
 * A second doc line so a later --doc render has more than one line.
 */
fun value(): Int {
    return 10
}

fun render(): Int {
    return value() + 1
}

val top = PI
